src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -420,15 +414,12 @@ export default {
 
 	computed: {
 		isLastReadMessage() {
-			if (!this.nextMessageId) {
+			if (!this.nextMessageId || this.id === this.conversation?.lastMessage?.id) {
 				// never display indicator on the very last message
 				return false
 			}
```

Comment:
It looks unclear to me that we need both checks here... In which cases one of these checks might be not enough? 
Maybe add a short comment with these case descriptions?

Reply:
Oh, `nextMessageId` is the next message in a group of messages from one author, while `lastMessage` is the last message in the conversation.

Reply:
In this case, shouldn't `this.id === this.conversation?.lastMessage?.id` be enough?